source/includes/kubernetes/_k8_releases.md

```diff
@@ -362,3 +386,40 @@ Mandatory | &nbsp;
 Optional | &nbsp;
 ------- | -----------
 `values` <br/>*string* | YAML structured text that will overwrite the default values for the upgrade/installation of the chart.
+
+<!-- UNINSTALL RELEASE -->
+### Uninstall a release
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/k8s/anenvironment/releases/pspensieri/aerospike-1579797954&operation=uninstall"
+   -d "request_body"
+
+# Request body example
```

Comment:
The same comment, although im not sure if we should just go for consistency

It does seem like there's inconsistency in the documentation whether `# The above command returns JSON structured like this:` is inside or outside the code block, and if it is outside it seems like one '#' is probably not enough because the text is very large compared to that around it

Reply:
Good catch, I'll compare with the other docs as well

Reply:
Fixed the rogue headers in https://github.com/cloudops/cloudmc-api-docs/pull/86/commits/7cee492164ec7e3e4994afdc992e4ad5a4677f19